SolveSATWithGrover/Tests.qs

```diff
@@ -336,13 +323,13 @@ namespace Quantum.Kata.GroversAlgorithm {
     @Test("QuantumSimulator")
     operation T22_Oracle_Exactly1SAT_Test () : Unit {
         // General SAT instances
-        for (i in 1..10) {
-            let (nVar, problem) = Generate_SAT_Instance(3);
-            Message($"Testing exactly-1 3-SAT instance ({nVar}, {SATInstanceAsString(problem)})...");
+        for (i in 2..6) { // test for 2..6 variables
```

Comment:
I would move the "for 2 .. 6 variables" part of the comment to the previous line, so that there is no inline comment (and it looks more like task 1.6)

Reply:
Addressed in latest commit